### PR TITLE
mesa: update to 20.1.9

### PR DESCRIPTION
**Pull request by st3r4g**
Tested GL app (kitty) and VK app (vkcube) under sway on x86_64-musl

### CODE REVIEW

File: srcpkgs/mesa/template [hunk 1]
```diff
@@ -23,7 +23,7 @@ license="MIT, LGPL-2.1-or-later"
 homepage="https://www.mesa3d.org/"
```
Comment (sgn) [inline srcpkgs/mesa/template:23]: Would you mind checking if replace `libxml2-python` with `libxml2-python3` in `hostmakedepends` and replace Juan's name with something else, Orphan is a good candidate?

Comment (st3r4g) [reply srcpkgs/mesa/template:23]: it seems `mesa` doesn't even depend on it anymore, so I can just remove it. I think @q66 is the de-facto maintainer, I can put his name if he wants

Comment (biopsin) [reply srcpkgs/mesa/template:23]: I've tested builds against libxml2-python3 last 2-3 releases, 
have not noticed any breakage, well atleast locally on x86_64

Comment (q66) [reply srcpkgs/mesa/template:23]: feel free to add me

Comment (st3r4g) [reply srcpkgs/mesa/template:23]: it seems that it was originally employed in mapi/glapi and later removed:
https://gitlab.freedesktop.org/mesa/mesa/-/commit/2de70fe23f320ce4f559e37057fe07b7af99cf5a
I guess it's safe to remove then

Comment (q66) [reply srcpkgs/mesa/template:22]: we should add `libxml2-devel` to `makedepends` though, since `freedreno` seems to need it to build some components

Comment (st3r4g) [reply srcpkgs/mesa/template:22]: Yeah, I've seen it. And it checks for `lua-devel`, `libarchive-devel` and `gzip` too. We could add them all to the `_have_arm` section

Comment (q66) [reply srcpkgs/mesa/template:22]: nah, i'd just add it normally, there's no point in conditionalizing it

**Pull request merged**